### PR TITLE
Enable CD releases

### PR DESCRIPTION
**Pull request by dwnusbaum**
See https://www.jenkins.io/doc/developer/publishing/releasing-cd/ and https://github.com/jenkins-infra/repository-permissions-updater/pull/2688.

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,7 +9,8 @@
     </parent>
 
     <artifactId>jsch</artifactId>
-    <version>${revision}${changelist}</version>
+    <!-- TODO: Switch separator from '.' to '-' next time jsch dependency is updated. -->
+    <version>${revision}.${changelist}</version>
```
Comment (dwnusbaum) [inline pom.xml:13]: The current plugin version is 0.1.55.2. 0.1.55 is the version of jsch, and the `.2` suffix is for this plugin. Normally we use `-` to separate the library version from the plugin version for API plugins like this, but I do not know for sure whether `0.1.55.2` would be considered older than `0.1.55-60.vfa_d521b_a_1eb_e` and so I just kept using `.` for now. If `0.1.56` ever gets released we could safely switch to `-` at that time.

If anyone knows for sure that Maven and Jenkins and anything else we might care about would consider `0.1.55-60.vfa_d521b_a_1eb_e` newer than `0.1.55.2`, then we could switch right now.

Comment (jtnord) [reply pom.xml:13]: don't hold your breath.
https://sourceforge.net/p/jsch/mailman/jsch-users/thread/22608f9c-3308-ef6a-0849-0166194f0840%40mlink.com/#msg37027593

I would suggest migrating anything that requires this to something else instead like mina ssh

Comment (dwnusbaum) [reply pom.xml:13]: Yes I know it is essentially abandoned, I am just noting the version number discrepancy here. 

> I would suggest migrating anything that requires this to something else instead like mina ssh

Sure I agree, but I do not have time to work on that. I am just trying to get a release out for #17 to simplify BOM testing.

**Review by zregvart — APPROVED** [review]
LGTM

**Review by basil — APPROVED** [review]
Last plugin-specific hack in BOM left :smile: :tada: 

https://github.com/jenkinsci/bom/blob/d31b9d59605dfdc65a2e5add495a1ad69169d097/pct.sh#L69-L81

**Comment by basil** [comment]
Thank you very much!

**Pull request merged**